airflow/models/__init__.py

```diff
@@ -3085,6 +3094,7 @@ def __init__(
         self.partial = False
         self.on_success_callback = on_success_callback
         self.on_failure_callback = on_failure_callback
+        self.doc_md = doc_md
```

Comment:
Where did this change come from?

Reply:
Some of the example DAGs set `doc_md`, which was raising errors from `mypy`. And it looks like we check this attribute in `views.py`.